descriptors/ModuleDescriptor-template.json

```diff
@@ -2,6 +2,19 @@
   "id": "${artifactId}-${version}",
   "name": "Inventory Storage Module",
   "provides": [
+    {
+      "id": "item-storage-dereferenced",
+      "version": "0.1",
+      "handlers": [{
+        "methods": ["GET"],
+        "pathPattern": "/item-storage-dereferenced/items",
+        "permissionsRequired": ["inventory-storage.items.collection.get"]
+      },{
+        "methods": ["GET"],
+        "pathPattern": "/item-storage-dereferenced/items/{id}",
+        "permissionsRequired": ["inventory-storage.items.item.get"]
+      }]
+    },
```

Comment:
See general comments above.  The existing item storage endpoint has the capacity for retrieving item records by Id-is there a good reason this one shouldn't?  It was trivial to implement once I had the code for CQL searching worked out.

Reply:
> The existing item storage endpoint has the capacity for retrieving item records by Id-is there a good reason this one shouldn't? It was trivial to implement once I had the code for CQL searching worked out.

Ok. I tend to want to minimise the API surface area. I'm ok with leaving this in. I do think it has slightly complicated the implementation code (because the mapping and other behaviour is shared).